themes/default/layouts/shortcodes/install-go.html

```diff
@@ -1,3 +1,21 @@
 <p class="mt-4">
     Install <strong><a href="https://golang.org/doc/install" target="_blank">Go</a></strong>. <br/>
 </p>
+
+<div class="note note-info" role="alert">
+    <div class="note-heading">
+        <i class="fas fa-info-circle mr-1"></i>
+        <span>Note</span>
+    </div>
+    <div class="note-body">
+        <p>
+            Pulumi requires Go 1.16 or later. If you're using Linux, your distribution may not
+            provide an up to date version of the Go compiler. To check what version of Go you have
+            installed, <code>go version</code> will tell you:
+        </p>
+<pre><code>
+$ go version
+go version go1.17.1 darwin/arm64
+</code></pre>
```

Comment:
```suggestion
            provide an up to date version of the Go compiler. To check what version of Go you have
            installed, use:<code>go version</code>.
        </p>
```

Reply:
@indrora what would you think of removing the code snippet?

Reply:
This looks good